app/data_access/regulation_computation.py

```diff
@@ -31,26 +32,55 @@ class Meta:
         description="Type d'utilisateur dont la version est utilisée pour le calcul de dépassement de seuil",
     )
 
-    alerts = graphene.List(
-        RegulatoryAlertOutput,
-        description="Liste des alertes remontées par ce calcul",
+    regulation_checks = graphene.List(
+        RegulationCheckOutput,
+        description="Liste des seuils règlementaires calculés",
         unit=graphene_enum_type(UnitType)(
             required=False,
             description="Unité de temps de ces seuils règlementaires",
         ),
     )
 
-    def resolve_alerts(self, info, unit=None):
-        base_query = RegulatoryAlert.query.filter(
+    def resolve_regulation_checks(self, info, unit=None):
+        base_query = RegulationCheck.query
+        if unit:
+            base_query = base_query.filter(RegulationCheck.unit == unit)
+        regulation_checks = base_query.all()
+
+        if not regulation_checks:
+            return None
+
+        regulatory_alerts = RegulatoryAlert.query.filter(
             RegulatoryAlert.user_id == self.user.id,
             RegulatoryAlert.day == self.day,
             RegulatoryAlert.submitter_type == self.submitter_type,
         )
 
-        if unit:
-            base_query = base_query.filter(
-                RegulatoryAlert.regulation_check.has(
-                    RegulationCheck.unit == unit
-                )
-            )
-        return base_query.all()
+        regulation_checks_extended = []
+        for regulation_check in regulation_checks:
+            regulatory_alert = regulatory_alerts.filter(
+                RegulatoryAlert.regulation_check_id == regulation_check.id
+            ).one_or_none()
+            setattr(regulation_check, "alert", regulatory_alert)
+            regulation_checks_extended.append(regulation_check)
+
+        return regulation_checks_extended
+
+
+class RegulationComputationByDayOutput(graphene.ObjectType):
+    def __init__(self, day, regulation_computations):
+        self.day = day
+        self.regulation_computations = regulation_computations
+
+    day = graphene.Field(
+        graphene.Date,
+        description="Journée du groupe",
```

Comment:
:green_circle: Proposition de modification :
```suggestion
        description="Journée pour laquelle les seuils sont calculés (pour les calculs hebdomadaires, il s'agit du premier jour de la semaine en considérant qu'elle commence le lundi)",
```